pyproject.toml

```diff
@@ -195,6 +195,7 @@ show_error_codes = true
 [tool.black]
 line-length = 100
 preview = true
+required_version = "24.1.1"
```

Comment:
Is this necessary? If someone is doing local pycontrails development, they should be using the dev dependencies ... should I think of this as an extra guardrail?

Reply:
Yes - I think this is. This will ensure that as this version is bumped, the team gets alerted to re-run the dev-install to bump black.

Reply:
Unfortunately I don't think there's a similar guardrail for ruff---so somebody with an old dev environment could continue using a version of ruff that doesn't match the one specified in pyproject.toml.